pages/event-resources.jsx

```diff
@@ -0,0 +1,407 @@
+var React = require('react');
+var ImageTag = require('../components/imagetag.jsx');
+
+var LinksContainer = React.createClass({
+  render: function() {
+    return (
+      <div className="row">{this.props.children}</div>
+    );
+  }
+});
+var PageLinker = React.createClass({
+  render: function() {
+    return (
+      <div className="page-linker col-sm-3 col-md-3 col-lg-3">
+        <a href={this.props.href}>{this.props.head}</a>
+        {this.props.children}
+      </div>
+    );
+  }
+});
+var EventItem = React.createClass({
+  render: function() {
+    return (
+      <div className="event-item">
+        <div className="event-icon">
+          <div className="participants-number">{this.props.participants}</div>
+          <div className="participants-label">PARTICIPANTS</div>
+        </div>
+        <div className="event-content">
+          <div className="event-item-header">{this.props.head}</div>
+          <div className="event-item-subheader">{this.props.subHead}</div>
+          <p className="event-item-content">{this.props.content}</p>
+        </div>
+      </div>
+    );
+  }
+});
+var LogoAsset = React.createClass({
+  render: function() {
+    return (
+      <div className="logo-asset-container col-sm-4 col-md-4 col-lg-3">
+        <div className="logo-asset-header">{this.props.head}</div>
+        <ImageTag className="logo-asset-img"
+        width={200} height={200}
+        src1x={this.props.src1x} src2x={this.props.src2x}
+        alt={this.props.alt}/>
+        <div className="logo-asset-hover">
+          <div className="logo-asset-center">{this.props.children}</div>
+        </div>
+      </div>
+    );
+  }
+});
+var LogoAssetLink = React.createClass({
+  render: function() {
+    return (
+      <a className="logo-asset-link" href={this.props.href}><span className="ion ion-ios-copy"></span>{this.props.children}</a>
+    );
+  }
+});
+var RemixLink = React.createClass({
+  render: function() {
+    return (
+      <a className="remix-link" href={this.props.href}>Remix</a>
+    );
+  }
+});
+var Tabulator = React.createClass({
```

Comment:
Now that we're using react-bootstrap, we could potentially use its tabs for this, but it's up to you... That said, though, as per https://github.com/mozilla/teach.webmaker.org/issues/388#issuecomment-86063715 I thought for v1 that we were going to actually go with a non-JS approach, so that it's both accessible and printable?